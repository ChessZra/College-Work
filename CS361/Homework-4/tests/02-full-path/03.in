/usr/bin/wc tests/02-full-path/file1.txt
