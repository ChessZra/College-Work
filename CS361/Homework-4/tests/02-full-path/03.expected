cs361sh>    5  500 3354 tests/02-full-path/file1.txt
cs361sh> 
